src/commands.rs

```diff
@@ -3950,53 +3955,81 @@ fn is_fast_forward(repo: RepoRef, branch_name: &str, new_target_id: &CommitId) -
 
 fn cmd_branch(ui: &mut Ui, command: &CommandHelper, args: &BranchArgs) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
-    let branch_name = &args.name;
+    let branch_names: Vec<&str> = if args.delete || args.forget {
+        args.names
+            .iter()
+            .map(|branch_name| {
+                match workspace_command
+                    .repo()
+                    .view()
```

Comment:
Can the `&View` be assigned to a variable to reduce line wrapping here? Can even be moved out of the iteration, I think.